CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 1.5.2 (July 1, 2015)
+### general
+  - Plugin manager: Added validation and warning when updating plugins between major versions (#3383).
+  - Performance improvements: String interpolation is widely used in LS to create keys combining dynamic
+    values from extracted fields. Added a caching mechanism where we compile this template on first use
+    and reuse them subsequently, giving us a good performance gain in configs that do a lot of date 
+    processing, sprintf, and use field reference syntax (#3425).
+  - Added warning when LS is running on a JVM version which has known issues/bugs (#2547).  
+  - Updated AWS based plugins to v2 of AWS ruby SDK. This involves an update to s3-input, s3-output,
+    sqs-input, sns-output.
+
+### input
+  - Lumberjack: This input was not handling backpressure properly from downstream plugins and
```

Comment:
Let's add that the `max_clients` setting now has no effect because it is not needed due to this improved circuit breaker. Pending https://github.com/logstash-plugins/logstash-input-lumberjack/pull/32